<launch>
	<arg name="rvizconfig" default="$(find yolo6d_ros)/rviz/display.rviz" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

	<node name="onigiriPose" pkg="yolo6d_ros" type="yolo6d_ros.py" output="screen" ns = "onigiri">
	</node>
</launch>